packages/typegen/test/options.test.ts

```diff
@@ -30,7 +30,9 @@ beforeEach(async () => {
       j json,
       jb jsonb,
       j_nn json not null,
-      jb_nn jsonb not null
+      jb_nn jsonb not null,
+      d_p numeric(8),
+      d_ps numeric(8,4)
```

Comment:
How about 
```suggestion
      d_ps numeric(8, 4)
```
?

Reply:
As far as I know, postgres strips the whitespace from the type. But I don't see the harm in covering the whitespace case too.